### ISSUE TITLE
Implement channels

### ISSUE THREAD

**Issue by alexislecompte** [open-telemetry/opentelemetry-collector-contrib#35860]
### Component(s)

exporter/splunkhec

### Is your feature request related to a problem? Please describe.

Indexer acknowledgement requires the use of a channel id, which does not appear to be an option at the moment.

### Describe the solution you'd like

An extra optional parameter to specify the channel ID for the collector would most likely be sufficient.

### Describe alternatives you've considered

Not enabling/using indexer acknowledgement is the only option without channel ID.

**Comment by atoulme** [comment]
I don't think this is so simple. Do you have a specification explaining how channels work, what values are accepted and how to relay them?

**Comment by atoulme** [comment]
@alexislecompte at this point your feature request is not accepted. Please help understand more the scope of the work.
